third_party/proton/csrc/lib/Profiler/Cupti/CuptiPCSampling.cpp

```diff
@@ -15,10 +15,10 @@ namespace {
 
 uint64_t getCubinCrc(const char *cubin, size_t size) {
   CUpti_GetCubinCrcParams cubinCrcParams = {
-      .size = CUpti_GetCubinCrcParamsSize,
-      .cubinSize = size,
-      .cubin = cubin,
-      .cubinCrc = 0,
+      /*size=*/CUpti_GetCubinCrcParamsSize,
```

Comment:
The change is fine to me but I would like the assignment comments to be changed to

```
/*variable name=*/
```

by adding an equal sign

Reply:
Done

Reply:
Thanks for review!